DiamondLightSource/dodal
# Add total counts attenuation scan

Fixes[ #701](https://github.com/DiamondLightSource/python-artemis/issues/701)
Adds functionality to devices to allow for the attenuation_optimisation_plan

To test:
Check code changes are sensible alongside the changes in https://github.com/DiamondLightSource/python-artemis/pull/742
